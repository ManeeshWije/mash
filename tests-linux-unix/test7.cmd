ls *.c
ls *.h

ls -l Makefile.*

echo tests-linux-unix/test[123].cmd
