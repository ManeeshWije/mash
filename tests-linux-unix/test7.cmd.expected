getopt.c
mash_main.c
mash_run.c
mash_tokenize.c
mash_run.h
mash_tokenize.h
os_defs.h
-rw-r--r-- 1 maneesh maneesh 315 Nov  9 20:03 Makefile.Darwin
-rw-r--r-- 1 maneesh maneesh 315 Nov  9 20:03 Makefile.Linux
-rw-r--r-- 1 maneesh maneesh 312 Nov  9 20:03 Makefile.Windows
tests-linux-unix/test1.cmd tests-linux-unix/test2.cmd tests-linux-unix/test3.cmd
